profiles/common/modules/features/nexteuropa_editorial/includes/nexteuropa_editorial.tracking.inc

```diff
@@ -222,22 +212,45 @@ function _nexteuropa_editorial_tracking_wbm_validate($form, &$form_state) {
  * @see _nexteuropa_editorial_tracking_entity_validate()
  */
 function _nexteuropa_editorial_tracking_field_validate($element, &$form_state, $form) {
-  $value = '';
-  $field_name = $element['#field_name'];
-  if (isset($element['#value'])) {
-    $value = $element['#value'];
-  }
-  elseif (isset($element['value'])) {
-    // The callback is called twice for field with text format.
-    // In this case, the inserted value is stored differently.
-    $value = $element['value']['#value'];
-    $element = $element['value'];
+  $entity = $element['#entity'];
+  $current_wbm_state = FALSE;
+  if (isset($form_state['values']['workbench_moderation_state_new'])) {
+    $current_wbm_state = $form_state['values']['workbench_moderation_state_new'];
   }
 
-  unset($form_state['tracked_changed_detected'][$field_name]);
-  if (preg_match('/<span[^>]+class="[^"]*ice-[^>]+>/', $value)) {
-    // Storing field info in form_state to treat it later with the entity.
-    $form_state['tracked_changed_detected'][$field_name] = $element['#name'];
+  $is_to_check = _nexteuropa_editorial_tracking_is_saving_stop($entity, $current_wbm_state);
+  if ($is_to_check) {
+    $value = '';
+    if (isset($element['#value'])) {
+      $value = $element['#value'];
+    }
+    elseif (isset($element['value'])) {
+      // The callback is called twice for field with text format.
+      // In this case, the inserted value is stored differently.
+      $value = $element['value']['#value'];
+      $element = $element['value'];
+    }
+
+    $field_path = implode('][', $element['#parents']);
+    unset($form_state['tracked_changed_detected'][$field_path]);
+    if (preg_match('/<span[^>]+class="[^"]*ice-[^>]+>/', $value)) {
+      // Flag used in entity validate callback.
+      $form_state['tracked_changed_detected'][$field_path] = TRUE;
+
+      // Get the field label.
+      if (!empty($element['#title'])) {
+        $title = $element['#title'];
+      }
+      else {
+        // Multi value fields have an empty '#title'.
+        $info = field_info_instance($entity->entity_type, $element['#field_name'], $element['#bundle']);
+        $title = check_plain($info['label']);
+      }
+
+      $message = t('!title contains tracked changes that may not be saved.',
```

Comment:
As we have also check on the entity level which will give a nice description what you need to do, I suggest to make this message bit more generic and descriptive ex. 'contains tracking changes information.'